### ISSUE TITLE
IndexError: list index out of range

### ISSUE THREAD

**Issue by Frank-Dz** [thmoa/octopus#25]
Hi~ Thanks for your great work. 
When I ran the code, I got the following error. 
```
WARNING:tensorflow:From /home/frank/Desktop/mesh-py3/my_venv/lib/python3.5/site-packages/tensorflow/python/framework/op_def_library.py:263: colocate_with (from tensorflow.python.framework.ops) is deprecated and will be removed in a future version.
Instructions for updating:
Colocations handled automatically by placer.
Traceback (most recent call last):
  File "infer_single.py", line 89, in <module>
    main(args.weights, args.name, args.segm_dir, args.pose_dir, args.out_dir, args.opt_steps_pose, args.opt_steps_shape)
  File "infer_single.py", line 21, in main
    model = Octopus(num=len(segm_files))
  File "/home/frank/PycharmProjects/octopus/model/octopus.py", line 186, in __init__
    conv_l3 = GraphConvolution(32, tf_A[3], activation='relu', name='conv_l3', trainable=False)(shape_features)
  File "/home/frank/Desktop/mesh-py3/my_venv/lib/python3.5/site-packages/keras/engine/base_layer.py", line 457, in __call__
    output = self.call(inputs, **kwargs)
  File "/home/frank/PycharmProjects/octopus/graphconv/graphconvlayer.py", line 43, in call
    output = supports[0]
IndexError: list index out of range
```

Plus, can you provide more environment information for setting up?
For example, the python version, exact tensorflow-gpu and keras version. (Some parts of  tensorflow-gpu are now changed I think.)

Thanks in advance!

**Comment by thmoa** [comment]
see requirements.txt

**Comment by SunYangtian** [comment]
@Frank-Dz I had the same problem with you. I also use python3 and I'm not sure whether the python version is the cause of this problem. Have you solved that? Actually I was trying to install dirt with python2 at start, but failed.

**Comment by thmoa** [comment]
python 3 version: https://github.com/thmoa/octopus/pull/22

**Comment by SuwoongHeo** [comment]
I had the same problem and it is solved by applying changes in #22 with python 3.7, TF 2.3.
